components/strava/actions/update-activity/update-activity.mjs

```diff
@@ -0,0 +1,32 @@
+import strava from "../../strava.app.js";
+
+export default {
+  name: "Update Activity",
+  description: "Updates the given activity that is owned by the authenticated athlete. Requires activity:write. Also requires activity:read_all in order to update Only Me activities. See `https://developers.strava.com/docs/reference/`",
```

Comment:
You can remove "Requires activity:write. Also requires activity:read_all in order to update Only Me activities"

Reply:
Done